easybuild/toolchains/fft/fftw.py

```diff
@@ -39,15 +39,18 @@ class Fftw(Fft):
     """FFTW FFT library"""
 
     FFT_MODULE_NAME = ['FFTW']
+    FFTW_API_VERSION = ''
 
     def _set_fftw_variables(self):
 
-        suffix = ''
-        version = self.get_software_version(self.FFT_MODULE_NAME)[0]
-        if LooseVersion(version) < LooseVersion('2') or LooseVersion(version) >= LooseVersion('4'):
-            raise EasyBuildError("_set_fft_variables: FFTW unsupported version %s (major should be 2 or 3)", version)
-        elif LooseVersion(version) > LooseVersion('2'):
-            suffix = '3'
+        suffix = self.FFTW_API_VERSION
+        if not suffix:
+            version = self.get_software_version(self.FFT_MODULE_NAME)[0]
+            if LooseVersion(version) < LooseVersion('2') or LooseVersion(version) >= LooseVersion('4'):
+                raise EasyBuildError("_set_fft_variables: FFTW unsupported version %s (major should be 2 or 3)",
+                                     version)
+            elif LooseVersion(version) > LooseVersion('2'):
+                suffix = '3'
```

Comment:
this is the only change in existing files in this PR. It's meant to reduce code duplication by allowing `FujitsuFFTW` to derive from `Fftw`, and it seems innocuous to me (and eventually even useful for other fftw variants that don't follow the API version, as the AMD one did for a while), but I can also make `FujitsuFFTW` derive from `Fft` instead and duplicate most of `Fftw` there

Reply:
Thanks for pointing this out, I think this approach makes sense.